GVFS/GVFS.Installer/CreateMacInstaller.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+CONFIGURATION=$1
+if [ -z $CONFIGURATION ]; then
```

Comment:
Why are we allowing optional parameters to this script? The csproj always passes in all the params, and making them optional here just seems to increase the risk of a bug when someone edits the csproj. I'd prefer to blow up if the params aren't exactly right.

Reply:
removed optional.